src/esrs/aggregate.rs

```diff
@@ -111,79 +96,48 @@ pub trait AggregateManager: Identifier {
     }
 
     /// Persists an event into the event store - recording it in the aggregate instance's history.
-    async fn persist(
+    /// The store will also project the event.
+    ///
+    /// The policies associated to the store are run here. Override this function to change this behaviour.
+    async fn run_store(
```

Comment:
I think that rather than `run_store` i would call it something like `store_events`

Reply:
i would pick `store_events`. The underlying machinery (e.g. event handlers) are expected to happen in this context.